x-pack/plugins/actions/server/actions_client.test.ts

```diff
@@ -758,6 +758,76 @@ describe('create()', () => {
       `"This mySuperRadTestPreconfiguredId already exist in preconfigured action."`
     );
   });
+
+  it('throws when creating a system connector', async () => {
+    actionTypeRegistry.register({
+      id: '.cases',
+      name: 'Cases',
+      minimumLicenseRequired: 'platinum',
+      supportedFeatureIds: ['alerting'],
+      validate: {
+        config: { schema: schema.object({}) },
+        secrets: { schema: schema.object({}) },
+        params: { schema: schema.object({}) },
+      },
+      isSystemActionType: true,
+      executor,
+    });
+
+    await expect(
+      actionsClient.create({
+        action: {
+          name: 'my name',
+          actionTypeId: '.cases',
+          config: {},
+          secrets: {},
+        },
+      })
+    ).rejects.toThrowErrorMatchingInlineSnapshot(
+      `"System action creation is forbidden. Action type: .cases."`
+    );
+  });
+
+  it('throws when creating a system connector where the action type is not registered but a system connector exists in the in-memory list', async () => {
```

Comment:
How could this happen?  Doesn't seem like it matters that there's an existing `inMemoryConnector` of this type, which presumably would have had to been registered before hand.

Reply:
Yes, it is impossible for this to happen. I added it in case we introduce a bug in the future and somehow becomes possible. Extreme scenario though 🙂.